docs/source/pythonapi/examples/mgxs-part-iv.ipynb

```diff
@@ -519,9 +519,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we must specify the type of domain over which we would like the `Library` to compute multi-group cross sections. The domain type corresponds to the type of tally filter to be used in the tallies created to compute multi-group cross sections. At the present time, the `Library` supports \"material,\" \"cell,\" and \"universe\" domain types. In this simple example, we wish to compute multi-group cross sections only for each material andtherefore will use a \"material\" domain type.\n",
+    "Now we must specify the type of domain over which we would like the `Library` to compute multi-group cross sections. The domain type corresponds to the type of tally filter to be used in the tallies created to compute multi-group cross sections. At the present time, the `Library` supports \"material\" \"cell\", \"universe\", and \"mesh\" domain types. In this simple example, we wish to compute multi-group cross sections only for each material andtherefore will use a \"material\" domain type.\n",
     "\n",
-    "**Note:** By default, the `Library` class will instantiate `MGXS` objects for each and every domain (material, cell or universe) in the geometry of interest. However, one may specify a subset of these domains to the `Library.domains` property."
+    "**Note:** By default, the `Library` class will instantiate `MGXS` objects for each and every domain (material, cell, universe, or mesh cell) in the geometry of interest. However, one may specify a subset of these domains to the `Library.domains` property."
```

Comment:
This is an OCD comment, so feel free to ignore it: based on my understanding of your implementation, "domain" refers to the "mesh" while "subdomain" refers to "mesh cell." Is this true? If so, perhaps consider revising "mesh cell" here to simply "mesh" unless there is something other reason you have for this wording choice (again, nbd, but these subtle word choices may lead to confusion if we aren't consistent).